### PR TITLE
update light client

### PR DESCRIPTION
**Pull request by melekes**

### CODE REVIEW

File: relayer/verifier.go [hunk 1]
```diff
@@ -19,51 +21,65 @@ func (c *Chain) StartUpdatingLiteClient(period time.Duration) {
 	for ; true; <-ticker.C {
 		err := c.UpdateLiteDBToLatestHeader()
 		if err != nil {
-			c.logger.Error(err.Error())
```
Comment (melekes) [inline relayer/verifier.go:22]: logger does not yet exist.

Comment (cmwaters) [reply relayer/verifier.go:22]: Shall I add one?

Comment (melekes) [reply relayer/verifier.go:22]: please do

Comment (melekes) [reply relayer/verifier.go:22]: this one is pretty good https://github.com/uber-go/zap

Comment (cmwaters) [reply relayer/verifier.go:22]: what's wrong with the tendermint logger?

Comment (melekes) [reply relayer/verifier.go:22]: nothing wrong with it. just not sure if we can import and use it here... it's located in tendermint repo after all. probably ok to use it

Comment (cmwaters) [reply relayer/verifier.go:22]: Okay, I'm not really sure what stuff is needed to be logged in specific regards to the chain so I am going to hold off on implementing anything

**Comment by jackzampolin** [comment]
Thank you for this @melekes! Going to go ahead and merge.

**Pull request merged**